src/config.sh

```diff
@@ -222,7 +222,7 @@ function parse_config_options()
   fi
 
   # Default values
-  options_values['SCOPE']='local'
+  options_values['SCOPE']=''
```

Comment:
I see the reasoning behind setting this value default to empty, but are you sure that this does not breaks other parts of the code?

Reply:
yes